Dockerfile

```diff
@@ -1,12 +1,5 @@
-FROM jimmycuadra/rust:1.16.0
+FROM debian:jessie-slim
 
-RUN mkdir -p /rust/app
-WORKDIR /rust/app
+COPY ./target/x86_64-unknown-linux-gnu/release/linkerd-tcp /usr/local/bin/
```

Comment:
If we changed `./target/x86_64-unknown-linux-gnu/release/linkerd-tcp` to `/build/linkerd-tcp`, then:
1. We don't run the risk of picking up random things in the target directory not built from dockerize.
2. It would be easier to build images that use a debug build, since the builder has to choose what directory to map into build.

dockerize could be something like
```
TARGET_DIR="./target/x86_64-unknown-linux-gnu/release"
if [ -n "$LINKERD_DEBUG" ]; then
  TARGET_DIR="./target/x86_64-unknown-linux-gnu/debug"
fi
docker run --rm -v "${TARGET_DIR}:/rust/app/build"
```